public/controllers/management/components/management/status/status-stats.js

```diff
@@ -14,12 +14,24 @@ import React, { Component } from 'react';
 import { EuiStat, EuiFlexItem, EuiFlexGroup } from '@elastic/eui';
 
 import { connect } from 'react-redux';
+import { UI_ORDER_AGENT_STATUS } from '../../../../../../common/constants';
+import { agentStatusColorByAgentStatus, agentStatusLabelByAgentStatus } from '../../../../../../common/services/wz_agent_status';
 
 export class WzStatusStats extends Component {
   _isMounted = false;
   constructor(props) {
     super(props);
     this.state = {};
+    this.agentStatus = ['total', ...UI_ORDER_AGENT_STATUS].map(status => ({
+      color: status !== 'total' ? agentStatusColorByAgentStatus(status) : 'primary',
+      description: `${status === 'total' ? 'Total agents' : agentStatusLabelByAgentStatus(status)}`,
+      status
+    }));
+    this.agentStatus.push({
+      color: undefined,
```

Comment:
Dangerous to initialize something to undefined

Reply:
This causes indirectly that the component used to render the indicator uses the appropriate color depending on the selected theme in the plugin platform.